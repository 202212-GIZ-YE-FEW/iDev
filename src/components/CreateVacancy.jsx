import Button from "@/components/ui/Button";
import Router from "next/router";
import PageIntro from "@/components/PageIntro";
import { withTranslation } from "next-i18next";

const CreateVacancy = ({ t }) => {
    const handleClick = () => {
        const router = require("next/router").default;
        router.push({
            pathname: "/careers/create",
        });
    };

    return (
        <>
            (
            <div className='container pb-8'>
                <PageIntro
                    title={t("doYouHavevacancy")}
                    subtitle={t("doYouHavevacancyDescrption")}
                />
                <Button
                    content={t("addToCareersList")}
                    textTransform='uppercase'
                    filled='true'
                    size='large'
                    fontSize='text-lg md:text-xl lg:text-2xl'
                    radius='md'
                    onClick={handleClick}
                />
            </div>
        </>
    );
};
export default withTranslation("career")(CreateVacancy);
