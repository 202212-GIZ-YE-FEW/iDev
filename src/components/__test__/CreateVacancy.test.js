import renderer from "react-test-renderer";

import CreateVacancy from "../CreateVacancy";

it("renders correctly", () => {
    const tree = renderer.create(<CreateVacancy />).toJSON();
    expect(tree).toMatchSnapshot();
});
