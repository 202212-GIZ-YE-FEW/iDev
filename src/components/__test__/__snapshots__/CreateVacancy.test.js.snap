// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  "(",
  <div
    className="container pb-8"
  >
    <div
      className="mb-5 text-start"
    >
      <span
        className="font-normal block text-3xl md:text-4xl rtl:md:text-3xl lg:text-5xl rtl:lg:text-4xl uppercase break-words"
      >
        doYouHavevacancy
      </span>
      <p
        className="text-lg md:text-xl lg:text-2xl text-black/50"
      >
        doYouHavevacancyDescrption
      </p>
    </div>
    <div
      className="relative self-center"
    >
      <button
        className="transform hover:bg-yellow transition duration-500  hover:animate-bounce hover:scale-75 active:bg-cyan focus:outline-none focus:ring focus:ring-cyan
                font-normal whitespace-nowrap cursor-pointer  uppercase text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] py-[8px] font-medium bg-cyan text-black "
        disabled={false}
        onClick={[Function]}
      >
        addToCareersList
      </button>
    </div>
  </div>,
]
`;
